src/implementations/twig/components/blockquote/blockquote.story.js

```diff
@@ -40,10 +41,28 @@ const getArgTypes = () => {
         type: 'text',
       },
     },
+    show_image: {
+      name: 'show_image',
+      type: { name: 'boolean' },
+      defaultValue: false,
+      description: 'Show image',
+      table: {
+        type: { summary: 'boolean' },
+        defaultValue: { summary: 'false' },
```

Comment:
also this small thing, this is not a parameter in the template, so please don't set any default value in the table which might look confusing...